Remove "experimental" from full site editing docs (#37655)
WordPress/gutenberg: docs/how-to-guides/themes/create-block-theme.md

# Create a block theme

The purpose of this tutorial is to show how to create a block theme and help theme developers transition to full site editing. It is recommended that you first read the [block theme overview](/docs/how-to-guides/themes/block-theme-overview.md).

You will learn about the required files, how to combine templates and template parts, how to add presets for global styles, and how to add blocks and export the templates in the site editor.

> Block themes are releasing in WordPress 5.9 as part of the full site editing project.

Reviewer: Block themes work with older WordPress versions if Gutenberg is active, should we include that?

Author: I added a line for previously available in Gutenberg plugin 11_

> Block themes are releasing in WordPress 5.9 as part of the full site editing project, previously required Gutenberg plugin 11.0+ or newer.

Reviewer: As a non native English speaker, this sentence is difficult for me to read, it feels fragmented.
Perhaps
"Block themes require WordPress 5.9. If you are using an earlier WordPress version, you need to active Gutenberg 11.0 or newer to enable full site editing features."

Author: Updated and reworded. Let me know if that sounds better.

```

## Creating the templates and template parts

Before continuing, install and activate your theme.

There are several ways to create templates and template parts:

-   Manually, by creating HTML files containing block markup.
-   Using the site editor.
-   Using the template editing mode in the block editor.

> The fourth way is temporary and involves going to the Appearance menu > Templates, and is not recommended because of its limitations.

Reviewer: ```suggestion
```

This is the part that is incorrect, because the menu items have been removed.

Templates and template parts that have been created or edited in the site editor or template editing mode
are saved to the database as custom post types. To export them as theme files, follow these steps:

-   In the site editor, open the **More tools and options** menu.
-   Select the **Export** option to download a zip file containing the files. Unpack the files.
-   Copy the updated `index.html` file from `theme/templates/` to your theme's `templates` folder.

> -   Copy template part one and two from `theme/parts/` to your theme's `parts` folder.

Reviewer: ```suggestion
-   Copy template parts from `theme/parts/` to your theme's `parts` folder.
```

> -   Rename the template parts to `header.html` and `footer.html`, respectively.

Reviewer: I did a quick test of the export in 5.9 RC 1 and learnt that the renaming is no longer needed, this bug has been fixed.
```suggestion
```

> -   Open `index.html` and update the template part slugs in the block markup.

Reviewer: Same thing, the slugs do not need to be munally updated any more.

> -   Open `index.html` and update the template part slugs in the block markup.

Reviewer: Same thing, the slugs do not need to be manually updated any more.
```suggestion
```

> Saved templates have precedence over theme files. To use the updated theme files, go to **Appearance > Templates** and
> **Appearance > Template parts** and delete the saved templates.

Reviewer: Appearance > templates was removed so this is not correct anymore.

I think that most of this passage can be removed and a link to the new site editor article can be added. https://wordpress.org/support/article/site-editor/

```suggestion
Saved templates have precedence over theme files.

To learn more about the Site Editor, see the [support article](https://wordpress.org/support/article/site-editor/)
```

Start by adding two curly brackets to the file:

```json
{}
```

Add the version number for the theme.json format. For Gutenberg 10.6, the version number is 1:

```json
{

> 	"version": 1

Reviewer: All versions should be updated to 2.

Author: Updated 👍

### Template parts

In the templeParts section, assign the two template parts that you created to their template areas.

> Add two keys: -`name`, the file name of the template part file without the file extension, -`area`, the name of the template area.

Reviewer: Add the third key, which is is `title`, the visible name in the editor.

Author: Added
